src/main/java/cn/nukkit/Server.java

```diff
@@ -123,7 +123,7 @@
 
     private Config whitelist;
 
-    private AtomicBoolean isRunning = new AtomicBoolean(true);
+    private boolean isRunning = true;
```

Comment:
These changes would cause concurrency visibility issues on multi-core systems. Information about this topic:
https://stackoverflow.com/questions/42868163/does-atomic-variables-guarantee-memory-visibility
https://stackoverflow.com/search?q=java+concurrency+visibility
https://www.logicbig.com/tutorials/core-java-tutorial/java-multi-threading/visibility-volatile.html
https://dzone.com/articles/3-tips-for-volatile-fields-in-java
http://jeremymanson.blogspot.com/2007/08/atomicity-visibility-and-ordering.html